src/FixedDataTableRoot-test.js

```diff
@@ -33,16 +33,32 @@ describe('FixedDataTableRoot', function() {
     });
   });
 
-  describe('initial render', function() {
-    const renderTable = (optionalProps = {}) => {
-      let table = (
+  class TestTable extends React.Component {
+    constructor(props) {
+      super(props);
+
+      this.state = {...props}
+    }
+
+    /**
+     * Returns state from FDT.Table object
+     *
+     * @return {!Object}
+     */
+    getTableState() {
+      return this.refs['table'].state;
```

Comment:
We use it to inspect the inner state for the tests. I'll update the annotation to further clarify